imports/collections/schemas/orders.js

```diff
@@ -456,6 +456,10 @@ export const Order = new SimpleSchema({
     optional: true
   },
   "history.$": History,
+  "language": {
```

Comment:
I think this should be more specifically named, maybe `ordererPreferredLanguage`?